README.rst

```diff
@@ -99,6 +99,10 @@ Caveats
   or datestamp and re-load the file. There is scope to do this automatically in
   future.
 
+  * If you have existing data, you can pre-populate the Data Dictionary:
+
+     paster --plugin=ckanext-xloader migrate_types -c /etc/ckan/default/ckan.ini
```

Comment:
Would you mind moving this to the Upgrading from DataPusher section, say before step 2?

Reply:
Can do, probably on Monday.